integration/volume_create_test.go

```diff
@@ -0,0 +1,141 @@
+package integration
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/http/httputil"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/stretchr/testify/require"
+)
+
+var _ = suite("compute/volume/create", func(t *testing.T, when spec.G, it spec.S) {
+	var (
+		expect   *require.Assertions
+		cmd      *exec.Cmd
+		baseArgs []string
```

Comment:
Why are the `baseArgs` declared but not initialized here? I don't think we're getting anything out of re-assigning them on every `Before` invocation, so it's probably more readable if they're up here.